lakefile.lean

```diff
@@ -4,7 +4,8 @@ open Lake DSL System
 package proofwidgets where
   preferReleaseBuild := true
   buildArchive? := "ProofWidgets4.tar.gz"
-  releaseRepo := "https://github.com/leanprover-community/ProofWidgets4"
+  releaseRepo := "https://github.com/mars0i/ProofWidgets4"
+  -- releaseRepo := "https://github.com/leanprover-community/ProofWidgets4"
```

Comment:
Please undo this change on this branch (for experimenting with different cloud release links, I would recommend having another branch which is not a PR branch).